bindings/build.gradle

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+def withoutLogger = { exclude group: 'org.slf4j', module: 'slf4j-log4j12'
+    exclude group: 'org.slf4j', module: 'slf4j-simple' }
+
+    dependencies {
```

Comment:
This change has nothing to do with reducing the time of tests, you are trying to bring the dependencies exclusive to bindings closer to them, is that correct?

Reply:
Yes. This is correct. Now that we are separating gradle files for bindings and core, looked like a right time to make the changes.